File: Electrodomestico.java
public class Electrodomestico extends Producto {
    String tipoDeAlimentacion;

    public Electrodomestico(String nombre, int numSerie, double price, String fechaImportacion, int stock,
            String tipoDeAlimentacion) {
        super(nombre, numSerie, price, fechaImportacion, stock);
        this.tipoDeAlimentacion = tipoDeAlimentacion;
    }
    
}

File: Electrodomestico.py
from Producto import Producto

class Electrodomestico(Producto):
    def __init__(self, nombre, numSerie, price, fechaImportacion, stock, tipoDeAlimentacion):
        super().__init__(nombre, numSerie, price, fechaImportacion, stock)
        self.tipoDeAlimentacion = tipoDeAlimentacion